behave/features/specialist_provider.feature

```diff
@@ -34,4 +32,66 @@ Scenario: Specialist Provider Accepts a case
   And I can see a 'Case accepted successfully' message
   When I return to the specialist provider cases dashboard page
   And I select the Accepted tab
-  Then I can see my accepted case reference number
\ No newline at end of file
+  Then I can see my accepted case reference number
+
+@wip-said
+Scenario: Specialist Provider Accepts a case
+  Given that I am viewing a case that I have accepted as a specialist provider
+  And I select the Legal help form
+  And The legal help form Your Details section has the values
```

Comment:
This can use the repeated step as well.

Reply:
The Your details section is a bit more complex in terms of layout. Some rows have 2 elements and some have three.
<img width="1078" alt="Screenshot 2022-05-04 at 10 49 21" src="https://user-images.githubusercontent.com/45761276/166659578-b2966e59-56dc-4cea-94e4-785c4601ea0e.png">

Reply:
ok, so keep separate.